packages/toolkit/src/dao/new.js

```diff
@@ -28,7 +28,7 @@ export default async function newDao(
   let template
```

Comment:
Perhaps we could rename the `new.js` file to `newDao.js` so it reflects the function name.